libs/movex/src/lib/movex-store/MemoryMovexStore.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { tillNextTick } from '../../specs/util/misc';
 import { MemoryMovexStore } from './MemoryMovexStore';
-require('console-group').install();
+import 'console-group';
```

Comment:
Hey @Abhishek-Mallick, this still needs to call install() I would say, or be removed at all since it's a test and the logs were used just for dev.